### DISCUSSION TITLE
Use of fontawesome?

### DISCUSSION THREAD

**Discussion by balazslevi** [gotenberg/gotenberg#523] [Q&A] [answered]
Can fontawesome be imported used in the provided html file, to be converted to pdf?
As far as I see it won't render them properly..

**Comment by gulien** [comment, marked as answer]
Hello @balazslevi,

It should work, either by sending directly the fonts or via CDN. For instance, these test data load a « local » font but also Google fonts: https://github.com/gotenberg/gotenberg/tree/main/test/testdata/chromium/html/sample1.

It should be pretty much the same with Fontawesome.

**Comment by seyfcom** [comment]
Hi ! Is ther any news, how to use Fontawesome in a html  converted in pdf  ? I tried a lot of things, nothing works except including directly the SVG themself, but that is really not great.
The example provided by @gulien  is not available anymore
Thanks !
